npm/src/upload/upload-form.ts

```diff
@@ -156,9 +156,11 @@ export class UploadForm {
         fileList,
```

Comment:
This file is getting pretty large. Do you think it can be split up a bit. Maybe not for this task, unless its straight forward.

Reply:
Hmm, I could move the extraction of files from the dragEvent code at the top and maybe the display message code at the bottom; I'll have a look.